tests/Hedgehog.Tests/PropertyTests.fs

```diff
@@ -0,0 +1,16 @@
+﻿module Hedgehog.Tests.PropertyTests
+
+open Hedgehog
+open Expecto
+open TestDsl
+
+let propertyTests = testList "Property tests" [
+    fableIgnore "generated C# list of five elements is not abbreviated in the failure report" <| fun _ ->
+        let prop = property {
+            let! xs = Range.singleton 0 |> Gen.int |> Gen.list (Range.singleton 5) |> Gen.map ResizeArray
+            return false
+        }
+        let report = Property.renderWith (PropertyConfig.withShrinks 0<shrinks> PropertyConfig.defaultConfig) prop
```

Comment:
```suggestion
        let report = property {
            let! xs = Range.singleton 0 |> Gen.int |> Gen.list (Range.singleton 5) |> Gen.map ResizeArray
            return false
        }
        |> Property.renderWith (PropertyConfig.withShrinks 0<shrinks> PropertyConfig.defaultConfig)
```

Reply:
It looks like putting parentheses around the RHS of the let binding is required for this to compile. Do you think it looks better still?

Reply:
How about that?

Reply:
Done